src/api/.rubocop.yml

```diff
@@ -93,14 +93,18 @@ Style/NumericLiteralPrefix:
 Style/RedundantSelf:
   Enabled: true
 
-# checks for space after `!`  
+# Checks for space after `!`
 Style/SpaceAfterNot:
   Enabled: true
 
 # Checks for the presence of parentheses around ternary conditions
 Style/TernaryParentheses:
   Enabled: true
 
+# Checks for unwanted parentheses in parameterless method calls
+Style/MethodCallParentheses:
```

Comment:
They are supposed to be alphabetically ordered, although it seems that they are not ordered anymore. :sob:

We can order it in another PR as it is unrelated to this PR. :wink: